### ISSUE TITLE
[Accessibility] nav tabs : tabpanel focus issues

### ISSUE THREAD

**Issue by cfagno** [twbs/bootstrap#31345]
When you select a tab the current `tabpanel` is not focusable (bootstrap v5).
With some screen readers (like NVDA, ORCA) you can't read the `tabpanel` content selected like WAI ARIA practices.
See https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-2/tabs.html : 
> When the `tab` list contains the focus, moves focus to the next element in the `tab` sequence, which is the `tabpanel` element.

**Comment by mdo** [comment]
Feel like it's focused for me? /cc @patrickhlauke

**Comment by patrickhlauke** [comment]
to clarify, the issue here is two-fold:

> When the tab list contains the focus, moves focus to the next element in the tab sequence, which is the tabpanel element.

bootstrap's tab list does not "contain the focus". the tab list is not a single tab stop, each individual tab is a separate tab stop.

however, NVDA and the like suppress regular reading keys (cursor keys) when you're focused on an element with `role="tab"`, under the assumption that it's part of a tab list that processes cursor keys to change tabs. they arguably shouldn't, but they do.

so, while bootstrap's tab list does not contain focus, and does not use cursor keys to focus (for tabs that follow the manual activation pattern) or switch (for tabs that follow the automatic activation pattern) tabs - and therefore shouldn't need tab panels themselves to be focusable - NVDA and co. probably require it as otherwise they become non-functional  unless users realise they have to jump explicitly back into reading mode (NVDA key + SPACE).

authors can work around this currently by explicitly adding `tabindex="0"` to each tab panel container ... but that makes them cumbersome for non-AT keyboard users. it's an annoying conundrum.

note that the bootstrap dynamic tabs / ARIA tabs approach is also currently only ... half-hearted. there's some work I started a while ago to remedy this, but time and lack of JS skills got in the way ... https://github.com/twbs/bootstrap/pull/28927

**Comment by patrickhlauke** [comment]
Noting that this https://github.com/twbs/bootstrap/pull/33079 should resolve this issue now as well.
